website/content/docs/audit/index.mdx

```diff
@@ -89,6 +89,57 @@ using the [secrets enable](/vault/docs/commands/secrets/enable) command, and the
 
 [auth tune](/vault/docs/commands/auth/tune)
 
+## Audit request headers
+
+Vault Operators can audit specific HTTP headers when those headers are present in
+the incoming client request. Configuration for audit request headers is applied via
+the [Vault API](/vault/api-docs/system/config-auditing).
+
+When Vault Operators [create](/vault/api-docs/system/config-auditing#create-update-audit-request-header)
+an exception allowing a request header to appear in the audit log, by default the
+value of the header is **not** HMAC'd, this is [configurable](/vault/api-docs/system/config-auditing#hmac)
+per header.
+
+### Default headers
+
+<Warning title="Behavior changes">
+
+  Prior to Vault 1.17 **only** headers configured by the Vault Operator could appear
+  in an audit log.
+
+  As of Vault 1.17 this behavior changes, see below for further information.
+
+</Warning>
+
+As of Vault 1.17 the following headers are **always automatically enabled** in
+order to facilitate correlation of requests across distributed systems which
+include Vault.
+
+- `correlation-id`
+- `x-correlation-id`
+
+By default, the values of these headers are **not** [HMAC'd](/vault/api-docs/system/config-auditing#hmac),
+however Vault Operators may override this setting by [manually configuring](/vault/api-docs/system/config-auditing#create-update-audit-request-header)
+the headers described above, supplying an `HMAC` value of `true`.
+
+#### Example override for `correlation-id`
```

Comment:
```suggestion
To ensure Vault uses HMAC on the header values during logging, set the `hmac` value to true for the `config/auditing/request-headers` API call.

For example, to enable HMAC for `correlation-id`
```

There's no need to create explicit settings for examples that illustrate something discussed in the paragraph above

Reply:
Ok that makes sense, the reason I added the two examples was just to reiterate that it's not just x-correlation-id but also correlation-id, which I'd imagine a user will want to configure both if they want HMAC to happen.